Gateway/Http/Client/TransactionCapture.php

```diff
@@ -125,10 +125,14 @@ private function placeMultipleCaptureRequests($service, $requestContainer, $requ
                 $singleResponse = $this->copyParamsToResponse($singleResponse, $request);
                 $response[self::MULTIPLE_AUTHORIZATIONS][] = $singleResponse;
             } catch (AdyenException $e) {
+                $pspReference = isset($request[OrderPaymentInterface::PSPREFRENCE]) ?
+                    $request[OrderPaymentInterface::PSPREFRENCE] :
+                    'pspReference not set';
+
                 $message = sprintf(
                     'Exception occurred when attempting to capture multiple authorizations.
                     Authorization with pspReference %s: %s',
-                    $request[OrderPaymentInterface::PSPREFRENCE],
+                    $pspReference,
```

Comment:
It could be more simpler to use `??` operator

```suggestion
                    $request[OrderPaymentInterface::PSPREFRENCE] ?? 'pspReference not set',
```